app/src/main/java/chat/rocket/android/members/ui/MembersFragment.kt

```diff
@@ -109,18 +109,38 @@ class MembersFragment : Fragment(), MembersView {
         ui { showToast(message) }
     }
 
+    override fun showInviteUsersButton() {
+        ui { text_invite_users.isVisible = true }
+    }
+
+    override fun hideInviteUserButton() {
+        ui { text_invite_users.isVisible = false }
+    }
+
     override fun showGenericErrorMessage() = showMessage(getString(R.string.msg_generic_error))
 
     private fun setupRecyclerView() {
         ui {
-            recycler_view.layoutManager = LinearLayoutManager(context)
+            val linearLayoutManager = LinearLayoutManager(context, RecyclerView.VERTICAL, false)
+            recycler_view.layoutManager = linearLayoutManager
             recycler_view.addItemDecoration(
                 DividerItemDecoration(
                     it,
                     DividerItemDecoration.HORIZONTAL
                 )
             )
+            endlessRecyclerViewScrollListener =
+                object : EndlessRecyclerViewScrollListener(linearLayoutManager) {
+                    override fun onLoadMore(
+                        page: Int,
+                        totalItemsCount: Int,
+                        recyclerView: RecyclerView
+                    ) {
+                        presenter.loadChatRoomsMembers(chatRoomId, page * 60L)
```

Comment:
What is the reason of passing the offset here?

Reply:
Same pattern as in ChatRoomFragement
https://github.com/RocketChat/Rocket.Chat.Android/blob/develop/app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt#L810

Reply:
I have simplified it (based on other classes we have too - e.g `MentionsPresenter.kt`), but I see your point by choosing this approach. But since the offset is used on the presenter, moving it there can keep the scope...there. But I see your point.